Resources/Prototypes/Recipes/Lathes/electronics.yml

```
@@ -777,4 +777,10 @@
   parent: BaseCircuitboardRecipe
   icon: { sprite:  Structures/Machines/holopad.rsi, state: base } # Goobstation
   id: HolopadMachineCircuitboard
-  result: HolopadMachineCircuitboard
\ No newline at end of file
+  result: HolopadMachineCircuitboard
+
+# Assmos - /tg/ gases
+- type: latheRecipe
+  parent: BaseCircuitboardRecipe
+  id: ElectrolyzerMachineCircuitboard
+  result: ElectrolyzerMachineCircuitboard
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)